specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2280 @@
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "polarization": {
+          "type": "string",
+          "description": "Polarization. e.g. (RHCP, LHCP).",
+          "enum": [
+            "RHCP",
+            "LHCP",
+            "linearVertical",
+            "linearHorizontal"
+          ],
+          "x-ms-enum": {
+            "name": "polarization",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "authorizations": {
+          "type": "array",
+          "description": "Authorized Ground Stations",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/AuthorizedGroundstation"
+          },
+          "x-ms-identifiers": []
+        }
+      },
+      "description": "List of authorized spacecraft links per ground station and the expiration date of the authorization."
+    },
+    "Spacecraft": {
+      "type": "object",
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SpacecraftsProperties",
+          "description": "Spacecraft Properties",
+          "allOf": [
+            {
+              "$ref": "#/definitions/SpacecraftsProperties"
+            }
+          ]
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Customer creates a spacecraft resource to schedule a contact."
+    },
+    "ResourceReference": {
+      "type": "object",
+      "required": [
+        "id"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource ID."
+        }
+      },
+      "x-ms-azure-resource": true,
+      "description": "Resource Reference."
+    },
+    "EndPoint": {
+      "type": "object",
+      "required": [
+        "ipAddress",
+        "endPointName",
+        "port",
+        "protocol"
+      ],
+      "properties": {
+        "ipAddress": {
+          "type": "string",
+          "description": "IP Address."
+        },
+        "endPointName": {
+          "type": "string",
+          "description": "Name of an end point."
+        },
+        "port": {
+          "type": "string",
+          "description": "TCP port to listen on to receive data."
+        },
+        "protocol": {
+          "type": "string",
+          "description": "Protocol either UDP or TCP.",
+          "enum": [
+            "TCP",
+            "UDP"
+          ],
+          "x-ms-enum": {
+            "name": "protocol",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Customer End point to store/retrieve data during a contact."
```

Comment:
>Customer End point to store/retrieve data during a contact." [](http://example.com/codeflow?start=22&length=60)

suggestion:
"Customer endpoint to store and retrieve data during a contact." (or should it be about a contact? what kind of contacts are these? people?)

Reply:
these are antenna-ground station contacts. We can make it a little more clear